### DISCUSSION TITLE
Error "class [auth] does not exist" after upgrade from 6.1.1 to 6.1.16

### DISCUSSION THREAD

**Discussion by nikiie8** [firefly-iii/firefly-iii#8905] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

After the update Firefly from 6.1.1 to 6.1.16 on self-hosted server when I run command any php artisan ... program I get the error.
In Container.php line 906:
Target class [auth] does not exist.
In Container.php line 904:
Class "auth" does not exist

### Debug information

Firefly III - 500 Internal Server Error :(
Whoops! A database error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

SQLSTATE[42703]: Undefined column: 7 ERROR: column "user_group_id" does not exist LINE 1: ...pdated_at", "created_at" from "preferences" where "user_grou... ^ (Connection: pgsql, SQL: select "id", "user_id", "name", "data", "updated_at", "created_at" from "preferences" where "user_group_id" is null and "user_id" = 1 and "name" = locale limit 1)

This error occured in file /var/www/firefly-iii/vendor/laravel/framework/src/Illuminate/Database/Connection.php on line 813 with code 42703.

### Expected behaviour

php artisan ... program should be executed without errors

### Steps to reproduce

I follow instructions from the website:
1. Download zip file
2. Check SHA sum
3. Move Firefly III folder to new place
4. Unzip file except storage folder
5. Copy storage folder and env file from old Firefly III folder to new one
6. Change ownership to www:data of new Firefly III folder and change access permission to 775 of storage subfolder
7. run php artisan migrate --seed command

### Additional info

I tried to unzip new Firefly version including and excluding storage folder.
Also I tried to run php artisan under sudo and w/o sudo. Nothing works.
When I open Firefly I got 500 error page

At the same time when I returned old Firefly 6.1.1. Everything works like a charm

**Comment by nikiie8** [comment]
Small update. I manage to fix php artisan error. Instead of copying old .env file, I used the new .env file and just add information about app and db connection from the old one. But still I see the 500 error, when I try to connect to Firefly  

Firefly III - 500 Internal Server Error :(
Whoops! An error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

An exception has been thrown during the rendering of a template ("file_put_contents(/var/www/firefly-iii/storage/framework/cache/data/e7/5f/e75fd100405decb4c2b5457bfa2767d35f7ff841): Failed to open stream: No such file or directory").

This error occured in file /var/www/firefly-iii/resources/views/index.twig on line 108 with code 0.

**Comment by nikiie8** [comment]
Fixed 500 error. By some reasons, some subfolders in storage folder had root ownership. As soon as I changed it back to www-data, everything have worked.

**Comment by JC5** [comment]
Thanks for sharing the resolution! I will definitely put this in the documentation!
